open3d_conversions/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
Lets update to 3.5, it will be needed for noetic and make the ROS2 port easier

Reply:
Done (5c4efbd)